### ISSUE TITLE
Migrate flutter_firebase_testlab bucket from flutter-infra to flutter-cirrus

### ISSUE THREAD

**Issue by keyonghan** [flutter/flutter#90680]
Firebase tests, running in flutter/plugins repo, are running under gcp project flutter-cirrus, but the test results are being uploaded to gcp project flutter-infra.

We need to migrate the bucket from flutter-infra to flutter-cirrus to avoid gcp projects boundaries crossing.

**Comment by godofredoc** [comment]
@ricardoamador  was this fixed by some of your changes?

**Comment by ricardoamador** [comment]
Yes I believe this was completed as part of my last okrs. I will close this.
